editor/state.js

```diff
@@ -12,6 +12,8 @@ import { reduce, keyBy, first, last, omit, without, flowRight } from 'lodash';
 import { combineUndoableReducers } from './utils/undoable-reducer';
 import effects from './effects';
 
+const isMobile = window.innerWidth <= 800;
```

Comment:
Should we tie this to one of our media query values, perhaps `< 782px` ? 

https://github.com/WordPress/gutenberg/blob/c650280/editor/assets/stylesheets/_variables.scss#L107